src/DataFiles.py

```diff
@@ -55,3 +56,20 @@ def getChannelIds() -> dict:
             "report_channels": config["report_channels"],  # list of int
             "leaderboard_channel": config["leaderboard_channel"],  # int
         }
+
+
+def getMMRMultiplier() -> float:
+    with open(configPath, "r") as config:
+        multiplier = json.load(config)["mmr_multiplier"]
+
+    return multiplier
+
+
+def updateMMRMultiplier(newMultiplier: float) -> float:
+    with open(configPath, "r") as configFile:
```

Comment:
You should be able to open the file with read and write permissions that way you don't have to `open` the file twice.

Reply:
Can you elaborate on this more? I'm not sure that I understand. There is a getMMRMultiplier() function and a updateMMRMultiplier() function that's very similar to the discord token methods just above for get and update.

Reply:
You're opening a file to read from it, closing it, and then re-opening the same file to write back to it. You should just be able to open the file once as read & write and do both operations in one `with` block. Something like:

```python
with open(configPath, "r+") as configFile:
   config = json.load(configFile)
   config["mmr_multiplier"] = newMultiplier
   json.dump(config, configFile, indent=2)
```

I see what you mean in that you modeled this off of how `updateDiscordToken` works, but maybe that should get updated too. Not sure if there was a specific reason I did it that way or not. Seems a bit wasteful to open the same file twice when you don't need to.

Reply:
Using "r+" generates the whole contents of the file again, inside the config file. It becomes stacked on itself. It doesn't change whats in there it just adds the contents again. Looking at how `with open()` works, it can only have exactly one of create/read/write/append

Reply:
Hmmmm. Okay. Keep it as it is then. We can always come back to it later.